browser/templates/browser/change_person.html

```diff
@@ -45,6 +51,20 @@
         {% endwith %}
     </div>
 </div>
+
+<!-- This is the changed_by field required by PersonModel -->
+{% with form.changed_by as field %}
+    <div class="form-group hidden">
+        {% for error in field.errors %}
+            <div class="alert alert-warning">{{ error }}</div>
+        {% endfor %}
+        <label>
+            {{ field }} {{ field.label }}
+        </label>
+        <p class="text-muted">{{ field.help_text }}</p>
+    </div>
+{% endwith %}
```

Comment:
just make the field hidden in the form then you need to print it here. also, why would a hidden field have (hidden) error messages?

Reply:
Made the form element through PersonForm